frontend/components/AddHostsModal/PlatformWrapper/PlatformWrapper.tsx

```diff
@@ -304,7 +325,104 @@ const PlatformWrapper = ({
     );
   };
 
+  const renderChromeLabel = (label: string, value: string) => {
+    const onCopyChromeLabel = (evt: React.MouseEvent) => {
+      evt.preventDefault();
+
+      stringToClipboard(value)
+        .then(() => setCopyMessage((prev) => ({ ...prev, [label]: "Copied!" })))
+        .catch(() =>
+          setCopyMessage((prev) => ({
+            ...prev,
+            [label]: "Copy failed",
+          }))
+        );
+
+      // Clear message after 1 second
+      setTimeout(
+        () => setCopyMessage((prev) => ({ ...prev, [label]: "" })),
+        1000
+      );
+
+      return false;
+    };
+
+    return (
+      <>
+        {label}
+        <span className="buttons">
+          <Button
+            variant="unstyled"
+            className={`${baseClass}__chrome-copy-icon`}
+            onClick={onCopyChromeLabel}
+          >
+            <Icon name="copy" />
```

Comment:
Maybe confirm with @rachaelshaw which copy icon we want? Looks like this one is different than in the[ figma](https://www.figma.com/file/hdALBDsrti77QuDNSzLdkx/%F0%9F%9A%A7-Fleet-EE-(dev-ready%2C-scratchpad)?type=design&node-id=16041%3A205158&t=Vz4h0TkNC3VhaOFE-1)

Reply:
Using the new one from the new style guide!